### PR TITLE
Disable added to select

### PR DESCRIPTION
**Pull request by ChaitanyaGadodia**

### CODE REVIEW

File: src/components/Select.tsx [hunk 1]
```diff
@@ -50,8 +51,10 @@ const Select: React.SFC<SelectProps> = props => {
             <div
               className={inputWrapper}
               onClick={() => {
-                toggleDropdown();
-                onDropdownToggle(isOpen);
+                if (!disabled) {
```
Comment (azizhk) [inline src/components/Select.tsx]: add onClick listener if !disabled.

File: src/components/Select.tsx [hunk 2]
```diff
@@ -65,6 +68,7 @@ const Select: React.SFC<SelectProps> = props => {
                 errorMessage={errorMessage}
                 readOnly
                 {...inputProps}
+                disabled={disabled}
```
Comment (azizhk) [inline src/components/Select.tsx]: Allow this to be overridable by inputProps

File: src/components/Select.tsx [hunk 3]
```diff
@@ -49,10 +50,14 @@ const Select: React.SFC<SelectProps> = props => {
...
-                toggleDropdown();
-                onDropdownToggle(isOpen);
-              }}
+              onClick={
+                (!disabled &&
+                  (() => {
+                    toggleDropdown();
+                    onDropdownToggle(isOpen);
+                  })) ||
+                undefined
```
Comment (ritz078) [inline src/components/Select.tsx]: use ternary here.

File: src/components/Select.tsx [hunk 4]
```diff
@@ -65,6 +70,7 @@ const Select: React.SFC<SelectProps> = props => {
                 errorMessage={errorMessage}
                 readOnly
                 {...inputProps}
+                disabled={disabled || (inputProps && inputProps.disabled)}
```
Comment (azizhk) [inline src/components/Select.tsx]: Just move `disabled={disabled}` above `{...inputProps}`

**Pull request merged**